CoreUtilities/test/unit/src/au/gov/asd/tac/constellation/utilities/gui/NotifyDisplayerNGTest.java

```diff
@@ -72,20 +72,26 @@ public static void tearDownClass() throws Exception {
         }
     }
 
-//    @Test
-//    public void display() {
-//        display(true, true, true);
-//        display(true, false, true);
-//        display(false, true, true);
-//        display(false, false, false);
-//    }
+    @Test
+    public void display() {    
+        if (!Boolean.TRUE.toString().equalsIgnoreCase(System.getProperty("java.awt.headless"))){
```

Comment:
are you able to add a todo comment to address this for headless? I think this is fine as a temporary solution but I think it might be better long term to try get the test workable in the headless environment as well